guidelines/index.html

```diff
@@ -783,24 +783,30 @@ <h1>Glossary</h1>
       
     	<section class="appendix" id="changelog">
     		<h2>Change Log</h2>
-    		<p>This section shows substantive changes made in WCAG 2.2 since WCAG 2.1. <a href="https://www.w3.org/WAI/WCAG21/errata/">Errata fixes to WCAG 2.1</a> have also been incorporated into WCAG 2.2.</p>
+    		<p>This section shows substantive changes incorporated into WCAG 2.2 since WCAG 2.1 and changes made since its original publication on 05 October 2023. <a href="https://www.w3.org/WAI/WCAG21/errata/">Errata fixes to WCAG 2.1</a> have also been incorporated into WCAG 2.2.</p>
```

Comment:
```suggestion
    		<p>This section shows substantive changes incorporated into WCAG 2.2 since WCAG 2.1 and changes made to 2.2 since its original publication on 05 October 2023. <a href="https://www.w3.org/WAI/WCAG21/errata/">Errata fixes to WCAG 2.1</a> have also been incorporated into WCAG 2.2.</p>
```